Update README with new script commands

sadobass opened the pull request:
Add more detailed explanation

sadobass committed:
Update README with new script commands

sadobass committed:
Update README with nuxtElysia settings

Add nuxtElysia configuration and update README

tkesgar commented on README.md:
```diff
@@ -54,13 +54,20 @@ export default defineNuxtConfig({
+    path: '/api',
+  },
+  nitro: {
+    preset: 'Bun'
```
Should be `bun`: https://nitro.build/deploy/runtimes/bun

tkesgar commented on README.md:
```diff
@@ -54,13 +54,20 @@ export default defineNuxtConfig({
 	  'nuxt-elysia'
-  ]
+  ],
+  nuxtElysia: {
```
Because the module options are optional, I think we can move this into *Module options* section below.

Probably something like this:

```
Module options

Use `nuxtElysia` to customize options for the `nuxt-elysia` module:

export default defineNuxtConfig({
  nuxtElysia: {
    path: '/my-api'
  },
})
```

In addition, use custom path as example instead of the default `/api`, so the user will be aware of the customization

tkesgar commented on README.md:
```diff
@@ -320,9 +327,9 @@ uses Node.js (if both Node.js and Bun are available). Therefore, you need to add
-    "dev": "bun --bun dev",
-    "build": "bun --bun build",
-    "preview": "bun --bun preview"
+    "dev": "nuxt --bun dev",
```
I am not aware of `nuxt --bun`, probably `bun --bun nuxt` instead?

sadobass replied:
‘dev’: ‘bun --bun dev’,
Configure package.json as above and
running `bun run dev` causes it to enter an infinite loop. (Image attached)
<img width="122" height="258" alt="스크린샷 2026-01-05 오전 9 58 56" src="https://github.com/user-attachments/assets/813e8a8c-dde4-40dc-9258-4e7b2fa87854" />

tkesgar replied:
Yeah `bun --bun dev` is wrong

The commands should be:

```
    "dev": "bun --bun nuxt dev",
    "build": "bun --bun nuxt build",
    "preview": "bun --bun nuxt preview",
```

`bun nuxt` runs the `nuxt` command, and `--bun` forces it to use Bun.

sadobass committed:
Update API path and scripts in README

Changed the /api route to /my-api-app
Changed “Bun” to lowercase
Changed the script command to bun --bun nuxt ...
